packages/slate-react/src/utils/clone-fragment.js

```diff
@@ -81,6 +81,17 @@ function cloneFragment(event, value, fragment = value.fragment) {
   }
 
   attach.setAttribute('data-slate-fragment', encoded)
+  
+  //  Builds plaintext object for clipboard line by line using top-level nodes.
+  //  Adds '\u000A' (unicode LF) to end of each line except the last.
+  //  This prevents line breaks from being lost when pasting to targets
+  //  which don't support html input.
+  let plainText = ''
+
+  for (let i = 0; i < contents.childNodes.length; i++) {
+    plainText += contents.childNodes[i].innerText
+    if (i < contents.childNodes.length - 1) plainText += '\u000A'
```

Comment:
Hey @Slapbox any reason you're using `'\u000A'` instead of just `'\n'`?

Reply:
No good one. We can change it to `\n`.

I just wanted to be extra-explicit for myself about what characters were in use while I was comparing the unicode outputs of Slate vs other major editors.